File: Python Exams/Basic Exam Collection/Basics Online Exam - 9 and 10 March 2019/05. Fitness Center/05. Fitness Center.py
gim_places = int(input())
visited, bought_products  = 0, 0
data_dict = {'back' : 0, 'chest' : 0, 'legs' : 0, 'abs' : 0, 'protein shake' : 0, 'protein bar' : 0}

for _ in range(gim_places):
    current_data = input().lower()
    data_dict[current_data] += 1

for k,v in data_dict.items():
    print(f'{v} - {k}')
    if k == 'back' or k == 'legs' or k == 'abs' or k == 'chest':
        visited += v
    else:
        bought_products += v

print(f'{visited / gim_places * 100:.2f}% - work out')
print(f'{bought_products / gim_places * 100:.2f}% - protein')


'''
7
Chest
Back
Legs
Legs
Abs
Protein shake
Protein bar

'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 9 and 10 March 2019/05. Fitness Center/05. Fitness Center.js
function fitnessCenter(arg) {
    let clients = Number(arg.shift())
    let back = `${arg.filter(x => x === 'Back').length}`
    let chest = `${arg.filter(x => x === 'Chest').length}`
    let legs = `${arg.filter(x => x === 'Legs').length}`
    let abs = `${arg.filter(x => x === 'Abs').length}`
    let proteinShake = `${arg.filter(x => x === 'Protein shake').length}`
    let proteinBar = `${arg.filter(x => x === 'Protein bar').length}`

    console.log(`${back} - back\n${chest} - chest\n${legs} - legs\n${abs} - abs\n${proteinShake} - protein shake\n${proteinBar} - protein bar`)
    console.log(`${((+back + +chest + +legs + +abs) / clients * 100).toFixed(2)}% - work out`)
    console.log(`${((+proteinShake + +proteinBar) / clients * 100).toFixed(2)}% - protein`)
}


// fitnessCenter([
//     10,
//     'Back',
//     'Chest',
//     'Legs',
//     'Abs',
//     'Protein shake',
//     'Protein bar',
//     'Protein shake',
//     'Protein bar',
//     'Legs',
//     'Abs',
// ])